scripts/devdeps.py

```diff
@@ -67,7 +71,9 @@ def depend_check(deps_name, *args):
         'pytest_cov',
         'pytest_selenium',
         'mock',
-        'websocket'
+        'websocket',
```

Comment:
I'm not sure this is used anymore. I think the tests all use `tornado.websocket` now